webex_bot/webex_bot.py

```diff
@@ -203,6 +203,8 @@ def process_incoming_message(self, teams_message, activity):
         # Remove the Bots display name from the message if this is not a 1-1
         if not is_one_on_one_space:
             raw_message = raw_message.replace(self.bot_display_name, '').strip()
+            # Also try to remove just the first name
+            raw_message = raw_message.replace(self.bot_display_name.split()[0], '').strip()
```

Comment:
Will this throw exception if bot only has a one word name since array[0] might not exist?

Reply:
Better to add a check to see if the `len(self.bot_display_name.split())` is at least 1 to be safe.